packages/wallets/src/components/Base/WalletText/WalletText.tsx

```diff
@@ -1,11 +1,12 @@
-import React from 'react';
+import React, { ComponentProps, CSSProperties, ReactNode } from 'react';
 import classNames from 'classnames';
 import styles from './WalletText.module.css';
 
-interface WalletTextProps {
+interface WalletTextProps extends ComponentProps<'span'> {
     align?: 'center' | 'left' | 'right';
-    children: React.ReactNode;
-    color?: 'black' | 'error' | 'general' | 'primary' | 'success' | 'warning' | 'white';
+    children: ReactNode;
+    color?: CSSProperties['color'] | 'error' | 'general' | 'primary' | 'success' | 'warning';
```

Comment:
@thisyahlen-deriv what about colors `black, blue, white` which still exist in scss file, please, add these colors to the type. Also, JFYI, I added red color with value `#ff444f` because I needed it for onboarding title

Reply:
Black, blue, white, red, purple and all generic colors will come from `CSSProperties['color']`. 😸

Reply:
@thisyahlen-deriv got it, didn't notice it, nice 👍